streaming/src/test/java/org/apache/spark/streaming/JavaWriteAheadLogSuite.java

```diff
@@ -17,13 +17,15 @@
 
 package org.apache.spark.streaming;
 
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.nio.ByteBuffer;
 import java.util.Arrays;
-import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
 
-import org.apache.commons.collections.CollectionUtils;
-import org.apache.commons.collections.Transformer;
+import com.google.common.base.Function;
```

Comment:
There were a few actual warnings fixed up here, but I think this file needed a fair bit of cleanup and I just did it:

- Get rid of single use of Commons Collections transformer methods in favor of Guava
- `.java` files almost never have two top level classes
- Always set character encoding when converting strings to/from bytes